### DISCUSSION TITLE
Request repository deletion

### DISCUSSION THREAD

**Discussion by ahnvilla** [community/community#82688] [Actions]
### Body

Hello, how can I request to delete a repository of another user because it is a code that I exposed from the company where I work?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by MEClouds** [comment]
To request the removal of a repository exposing company code:

1. **Contact GitHub Support:**
   Open a support ticket with GitHub, clearly explaining the situation. [Contact GitHub Support](https://support.github.com/contact) and provide details about the repository, its URL, and specify that it contains code from your company.

2. **Proof of Ownership:**
   Include evidence proving your association with the company and your authorization to represent them in this matter. This may include a company email or any relevant documentation.

3. **Infringement Details:**
   Clearly outline how the repository violates your company's rights. Specify the code snippets or files that shouldn't be publicly available and highlight the potential harm or risk.

4. **Urgency Request:**
   If the exposure poses an immediate threat, emphasize the urgency of the situation. GitHub support may expedite the process based on the severity of the infringement.

5. **Follow GitHub Procedures:**
   Respect GitHub's policies and procedures. Be concise in your communication, adhere to their [guidelines](https://docs.github.com/en/github/site-policy/github-terms-of-service), and await their instructions on the necessary steps.

By following these steps, you increase the likelihood of a swift and effective resolution to protect your company's code.
